2023-10-14 01:29:20,468 INFO    StreamThr :63426 [internal.py:wandb_internal():86] W&B internal server running at pid: 63426, started at: 2023-10-14 01:29:20.467958
2023-10-14 01:29:20,469 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: status
2023-10-14 01:29:20,477 INFO    WriterThread:63426 [datastore.py:open_for_write():85] open: /home/ubuntulsa/Sem5/SMAI/Assignment 3/wandb/run-20231014_012920-bg8fhymq/run-bg8fhymq.wandb
2023-10-14 01:29:20,478 DEBUG   SenderThread:63426 [sender.py:send():380] send: header
2023-10-14 01:29:20,478 DEBUG   SenderThread:63426 [sender.py:send():380] send: run
2023-10-14 01:29:21,686 INFO    SenderThread:63426 [dir_watcher.py:__init__():211] watching files in: /home/ubuntulsa/Sem5/SMAI/Assignment 3/wandb/run-20231014_012920-bg8fhymq/files
2023-10-14 01:29:21,686 INFO    SenderThread:63426 [sender.py:_start_run_threads():1122] run started: bg8fhymq with start time 1697227160.467176
2023-10-14 01:29:21,702 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: check_version
2023-10-14 01:29:21,703 DEBUG   SenderThread:63426 [sender.py:send_request():407] send_request: check_version
2023-10-14 01:29:21,910 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: run_start
2023-10-14 01:29:21,912 DEBUG   HandlerThread:63426 [system_info.py:__init__():32] System info init
2023-10-14 01:29:21,912 DEBUG   HandlerThread:63426 [system_info.py:__init__():47] System info init done
2023-10-14 01:29:21,912 INFO    HandlerThread:63426 [system_monitor.py:start():194] Starting system monitor
2023-10-14 01:29:21,912 INFO    SystemMonitor:63426 [system_monitor.py:_start():158] Starting system asset monitoring threads
2023-10-14 01:29:21,913 INFO    HandlerThread:63426 [system_monitor.py:probe():214] Collecting system info
2023-10-14 01:29:21,914 INFO    SystemMonitor:63426 [interfaces.py:start():190] Started cpu monitoring
2023-10-14 01:29:21,916 INFO    SystemMonitor:63426 [interfaces.py:start():190] Started disk monitoring
2023-10-14 01:29:21,916 INFO    SystemMonitor:63426 [interfaces.py:start():190] Started memory monitoring
2023-10-14 01:29:21,918 INFO    SystemMonitor:63426 [interfaces.py:start():190] Started network monitoring
2023-10-14 01:29:21,924 DEBUG   HandlerThread:63426 [system_info.py:probe():196] Probing system
2023-10-14 01:29:21,924 DEBUG   HandlerThread:63426 [system_info.py:probe():244] Probing system done
2023-10-14 01:29:21,924 DEBUG   HandlerThread:63426 [system_monitor.py:probe():223] {'os': 'Linux-6.2.0-34-generic-x86_64-with-glibc2.35', 'python': '3.10.12', 'heartbeatAt': '2023-10-13T19:59:21.924677', 'startedAt': '2023-10-13T19:59:20.455657', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'host': 'ubuntulsa-HP-Pavilion-Laptop-14-dv0xxx', 'username': 'ubuntulsa', 'executable': '/bin/python3', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 1162.201, 'min': 400.0, 'max': 4700.0}, 'cpu_freq_per_core': [{'current': 1272.509, 'min': 400.0, 'max': 4700.0}, {'current': 1235.523, 'min': 400.0, 'max': 4700.0}, {'current': 1359.935, 'min': 400.0, 'max': 4700.0}, {'current': 1317.295, 'min': 400.0, 'max': 4700.0}, {'current': 908.245, 'min': 400.0, 'max': 4700.0}, {'current': 1327.132, 'min': 400.0, 'max': 4700.0}, {'current': 1315.909, 'min': 400.0, 'max': 4700.0}, {'current': 1368.048, 'min': 400.0, 'max': 4700.0}], 'disk': {'/': {'total': 91.10977554321289, 'used': 14.47625732421875}}, 'memory': {'total': 15.38132095336914}}
2023-10-14 01:29:21,925 INFO    HandlerThread:63426 [system_monitor.py:probe():224] Finished collecting system info
2023-10-14 01:29:21,925 INFO    HandlerThread:63426 [system_monitor.py:probe():227] Publishing system info
2023-10-14 01:29:21,925 DEBUG   HandlerThread:63426 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2023-10-14 01:29:21,925 DEBUG   HandlerThread:63426 [system_info.py:_save_pip():68] Saving pip packages done
2023-10-14 01:29:21,925 INFO    HandlerThread:63426 [system_monitor.py:probe():229] Finished publishing system info
2023-10-14 01:29:21,926 DEBUG   SenderThread:63426 [sender.py:send():380] send: files
2023-10-14 01:29:21,926 INFO    SenderThread:63426 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2023-10-14 01:29:21,934 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: stop_status
2023-10-14 01:29:21,934 DEBUG   SenderThread:63426 [sender.py:send_request():407] send_request: stop_status
2023-10-14 01:29:21,984 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: internal_messages
2023-10-14 01:29:22,602 DEBUG   SenderThread:63426 [sender.py:send():380] send: telemetry
2023-10-14 01:29:22,687 INFO    Thread-12 :63426 [dir_watcher.py:_on_file_created():271] file/dir created: /home/ubuntulsa/Sem5/SMAI/Assignment 3/wandb/run-20231014_012920-bg8fhymq/files/wandb-metadata.json
2023-10-14 01:29:22,687 INFO    Thread-12 :63426 [dir_watcher.py:_on_file_created():271] file/dir created: /home/ubuntulsa/Sem5/SMAI/Assignment 3/wandb/run-20231014_012920-bg8fhymq/files/requirements.txt
2023-10-14 01:29:23,518 INFO    wandb-upload_0:63426 [upload_job.py:push():131] Uploaded file /tmp/tmp94xrc6kgwandb/atufc4d5-wandb-metadata.json
2023-10-14 01:29:25,603 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: status_report
2023-10-14 01:29:30,604 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: status_report
2023-10-14 01:29:35,605 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: status_report
2023-10-14 01:29:36,935 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: stop_status
2023-10-14 01:29:36,935 DEBUG   SenderThread:63426 [sender.py:send_request():407] send_request: stop_status
2023-10-14 01:29:36,976 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: internal_messages
2023-10-14 01:29:37,274 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:29:37,347 DEBUG   SenderThread:63426 [sender.py:send():380] send: history
2023-10-14 01:29:37,347 DEBUG   SenderThread:63426 [sender.py:send_request():407] send_request: summary_record
2023-10-14 01:29:37,349 INFO    SenderThread:63426 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-10-14 01:29:37,691 INFO    Thread-12 :63426 [dir_watcher.py:_on_file_created():271] file/dir created: /home/ubuntulsa/Sem5/SMAI/Assignment 3/wandb/run-20231014_012920-bg8fhymq/files/wandb-summary.json
2023-10-14 01:29:38,512 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:29:38,513 DEBUG   SenderThread:63426 [sender.py:send():380] send: history
2023-10-14 01:29:38,513 DEBUG   SenderThread:63426 [sender.py:send_request():407] send_request: summary_record
2023-10-14 01:29:38,514 INFO    SenderThread:63426 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-10-14 01:29:38,691 INFO    Thread-12 :63426 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/ubuntulsa/Sem5/SMAI/Assignment 3/wandb/run-20231014_012920-bg8fhymq/files/wandb-summary.json
2023-10-14 01:29:38,691 INFO    Thread-12 :63426 [dir_watcher.py:_on_file_created():271] file/dir created: /home/ubuntulsa/Sem5/SMAI/Assignment 3/wandb/run-20231014_012920-bg8fhymq/files/output.log
2023-10-14 01:29:40,692 INFO    Thread-12 :63426 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/ubuntulsa/Sem5/SMAI/Assignment 3/wandb/run-20231014_012920-bg8fhymq/files/output.log
2023-10-14 01:29:41,515 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: status_report
2023-10-14 01:29:46,516 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: status_report
2023-10-14 01:29:51,522 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: status_report
2023-10-14 01:29:51,935 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: stop_status
2023-10-14 01:29:51,936 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: internal_messages
2023-10-14 01:29:56,936 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:29:57,266 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:29:58,735 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:30:01,937 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:30:06,935 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: internal_messages
2023-10-14 01:30:11,939 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:30:14,725 INFO    SenderThread:63426 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/connection.py", line 203, in _new_conn
    sock = connection.create_connection(
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/util/connection.py", line 60, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 790, in urlopen
    response = self._make_request(
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 491, in _make_request
    raise new_e
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 467, in _make_request
    self._validate_conn(conn)
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 1092, in _validate_conn
    conn.connect()
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/connection.py", line 611, in connect
    self.sock = sock = self._new_conn()
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/connection.py", line 210, in _new_conn
    raise NameResolutionError(self.host, self, e) from e
urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x7ffb581aebc0>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/connectionpool.py", line 844, in urlopen
    retries = retries.increment(
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/urllib3/util/retry.py", line 515, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ffb581aebc0>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)"))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 58, in execute
    request = self.session.post(self.url, **post_args)
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/ubuntulsa/.local/lib/python3.10/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ffb581aebc0>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)"))
2023-10-14 01:30:16,940 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:30:17,539 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:30:19,004 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:30:21,918 DEBUG   SystemMonitor:63426 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2023-10-14 01:30:21,936 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: internal_messages
2023-10-14 01:30:26,942 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:30:31,943 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:30:36,936 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: internal_messages
2023-10-14 01:30:37,865 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:30:39,346 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:30:41,944 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:30:46,945 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:30:51,936 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: internal_messages
2023-10-14 01:30:56,946 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:30:58,294 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:30:59,764 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: partial_history
2023-10-14 01:31:00,767 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: poll_exit
2023-10-14 01:31:05,768 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:10,769 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:15,771 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:20,772 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:25,774 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:30,775 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:35,777 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:40,778 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:45,780 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:50,781 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:31:55,782 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:00,783 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:05,785 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:10,786 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:15,788 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:20,789 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:25,790 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:30,792 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:35,793 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:40,794 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:45,796 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:50,797 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:32:55,798 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:00,800 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:05,801 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:10,802 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:13,496 WARNING FileStreamThread:63426 [file_stream.py:request_with_retry():668] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Read timed out. (read timeout=180). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7ffb5a2ab9d0>>, timeout=180), args: ('https://api.wandb.ai/files/mourya/mnist-cnn-hyperparam-tuning/bg8fhymq/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2023-10-14 01:33:15,518 WARNING FileStreamThread:63426 [file_stream.py:request_with_retry():668] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mourya/mnist-cnn-hyperparam-tuning/bg8fhymq/file_stream (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ffb58072470>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)")). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7ffb5a2ab9d0>>, timeout=180), args: ('https://api.wandb.ai/files/mourya/mnist-cnn-hyperparam-tuning/bg8fhymq/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2023-10-14 01:33:15,804 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:19,647 WARNING FileStreamThread:63426 [file_stream.py:request_with_retry():668] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mourya/mnist-cnn-hyperparam-tuning/bg8fhymq/file_stream (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ffb581ae500>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)")). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7ffb5a2ab9d0>>, timeout=180), args: ('https://api.wandb.ai/files/mourya/mnist-cnn-hyperparam-tuning/bg8fhymq/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2023-10-14 01:33:20,805 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:25,807 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:29,351 WARNING FileStreamThread:63426 [file_stream.py:request_with_retry():668] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/mourya/mnist-cnn-hyperparam-tuning/bg8fhymq/file_stream (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7ffb581af8b0>: Failed to resolve 'api.wandb.ai' ([Errno -3] Temporary failure in name resolution)")). func: functools.partial(<bound method Session.post of <requests.sessions.Session object at 0x7ffb5a2ab9d0>>, timeout=180), args: ('https://api.wandb.ai/files/mourya/mnist-cnn-hyperparam-tuning/bg8fhymq/file_stream',), kwargs: {'json': {'complete': False, 'failed': False, 'dropped': 0, 'uploaded': []}}
2023-10-14 01:33:30,808 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:35,809 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:40,810 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:45,811 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:50,812 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:33:55,813 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:00,815 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:05,816 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:10,817 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:15,818 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:20,820 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:25,821 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:30,822 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:35,823 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:40,825 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
2023-10-14 01:34:45,826 DEBUG   HandlerThread:63426 [handler.py:handle_request():146] handle_request: keepalive
